Adyen/Components/Base/PaymentComponentData.swift

```diff
@@ -20,16 +20,24 @@ public protocol ShopperInformation {
 
 }
 
-/// Any object that holds shopper billing address information/
+/// Any object that holds shopper billing address information
 public protocol BillingAddressInformation {
 
     /// The billing address information.
     var billingAddress: AddressInfo? { get }
 
 }
 
+/// Any object that holds shopper social security number
+public protocol SocialSecurityNumberInformation {
+
+    /// The social security number information.
+    var socialSecurityNumber: String? { get }
```

Comment:
Should be non optional, right?

Reply:
yes, but because of the way `PaymentComponentData` is implementing this, it has to be optional. same with `ShopperInformation` and `BillingAddressInformation` unfortunately

Reply:
But `PaymentComponentData` doesn't really have to implement any of those interfaces 🤔
Those interfaces needed for `PaymentMethodDetails` implementation, so  PaymentComponentData could fetch corespondent data from it

Reply:
But PaymentComponentData doesn't really have to implement any of those interfaces 🤔
Those interfaces needed for PaymentMethodDetails implementation, so PaymentComponentData could fetch corespondent data from it

Reply:
`PaymentComponentData` implements them to be used in `PaymentsRequest`